straxen/rundb.py

```diff
@@ -295,3 +273,30 @@ def run_metadata(self, run_id, projection=None):
         if self.reader_ini_name_is_mode:
             doc['mode'] = doc.get('reader', {}).get('ini', {}).get('name', '')
         return doc
+
+
+def get_mongo_url(hostname):
```

Comment:
Was splitting this function out of the class strictly necessary? I could see if we were exporting it for other modules, but we aren't (at least there's no `@export`)

Reply:
We need it in the online_monitor. I don't like duplicating code so therefore I recon it's better to split it.